android-core/src/main/java/com/mparticle/modernization/identity/MParticleIdentity.kt

```diff
@@ -0,0 +1,94 @@
+package com.mparticle.modernization.identity
+
+import com.mparticle.identity.IdentityApiRequest
+import com.mparticle.identity.IdentityApiResult
+import com.mparticle.identity.MParticleUser
+import com.mparticle.modernization.MParticleCallback
+import com.mparticle.modernization.core.MParticleComponent
+import org.jetbrains.annotations.NotNull
+import org.jetbrains.annotations.Nullable
+
+interface MParticleIdentity : MParticleComponent {
+
+    /**
+     * Login
+     * @param callback
+     */
+    fun login(@NotNull callback: IdentityCallback)
+
+    /**
+     * Logout
+     * @param callback
+     */
+    fun logout(@NotNull callback: IdentityCallback)
+
+    /**
+     * Returns the user by id or the current user if [mpId] is null
+     * @param mpId if null returns the current user, if not the user by id
+     * @param callback
+     */
+    fun getUser(@Nullable mpId: Long?, @NotNull callback: IdentityCallback)
```

Comment:
in the Web SDK, we have the concept of `getCurrentUser` that I don't believe exists in Native. Is this something we should move away from on the Web SDK to be consistent, or Native also support this?

Reply:
This method as well as the one on line 43 return the current user if null is passed for `mpId`. So it has the functionality, just not as a dedicated method. Whether it would be better to have a dedicated method or to use this way of doing it can be up for debate, but I think we should standardize on one solution across platforms.

Reply:
Yup exactly its just an option, we should follow a standard, either more precise and granular api or less granular but with more attributes)